client/pull.go

```diff
@@ -164,7 +163,7 @@ func (c *Client) Pull(ctx context.Context, ref string, opts ...RemoteOpt) (_ Ima
 
 	if unpacker != nil && ur.Unpacks == 0 {
 		// Unpack was tried previously but nothing was unpacked
-		// This is at least required for schema 1 image.
+		// This was at least required for schema 1 image.
```

Comment:
nit: probably can just remove this line.

Reply:
reads fine I think  ... history good :)